network/tcp.go

```diff
@@ -0,0 +1,97 @@
+// Provides Network Stats (TCP and TCP6)
+package network
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/intelsdi-x/snap-plugin-collector-docker/mounts"
+	"github.com/intelsdi-x/snap-plugin-collector-docker/wrapper"
+)
+
+func TcpStatsFromProc(rootFs string, pid int) (wrapper.TcpStat, error) {
```

Comment:
exported function TcpStatsFromProc should have comment or be unexported. 

Consider changing `Tcp`/`tcp` parts in name of functions and variables into "TCP". This is the same case as it is with "ID" but I see that it is problematic for unexported functions so it is up to you.